UsageCollectorWorkerService.Tests/StartupArgumentValidatorTests/ValidateStartupArgumentsTest.cs

```diff
@@ -0,0 +1,106 @@
+using UsageCollectorWorkerService.Validators;
+using Xunit.Abstractions;
+
+namespace UsageCollectorWorkerService.Tests.StartupArgumentValidatorTests;
+
+public class ValidateStartupArgumentsTest : TestBase
+{
+    public ValidateStartupArgumentsTest(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [Theory]
+    [InlineData(new object[] {new string[] { "http://192.168.0.55:5000/api/resourceusage/save", "20", "5"}})]
+    public void ValidateStartupArguments_ValidDataEntered_NoExceptionThrown(string[] cliArguments)
+    {
+        //Arrange
+        //Act
+        StartupArgumentsValidator.ValidateStartupArguments(cliArguments);
+        //Assert
+    }
+
+    [Theory]
+    [InlineData(new object[] {new string[] { "http://192.168.0.55:5000/api/resourceusage/save", "20", "5", "6"}})]
+    [InlineData(new object[] {new string[] { "http://192.168.0.55:5000/api/resourceusage/save", "20"}})]
+    [InlineData(new object[] {new string[] { }})]
+    [InlineData(new object[] {null})]
+    public void ValidateStartupArguments_IncorrectNumberOfArguments_ArgumentExceptionThrown(string[] cliArguments)
```

Comment:
More common approach would be to have just one Theory with all InlineDatas and pass exception message as another argument. But then you would lose more descriptive names like IncorrectNumberOfArguments and have to move it to a comment for example. So this aproach is fine as well and it is to your personal preference.